### DISCUSSION TITLE
Issue Triage Sessions

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#2977] [Ideas]
Just gathering some thoughts here...

How would you all feel about removing Stalebot (the integration that auto-closes old issues) and moving to community events where we get together and triage issues manually?

Some ideas:
- We could do this on a live stream maybe?
  - Would be a great way to encourage contributors too we could assign issues to folks live on the stream
  - Might need to limit this option to once a month to fit Eddie's schedule?
- We could host regular voice calls in our Discord server?
  - This option would allow us to host these with more frequency, since any moderator could run them.
  - However, we'd be limited to 25 participants if we want video sharing.
  - Alternative would be something like Google Meet?

**Comment by suitupstan** [comment]
I'd like RadioEddieStage or a Discord VC for this 🤔 

Also, we had a convo about team-eddie members being able to create VCs the setup we make for it, can maybe help 🤔

**Reply by naomi-lgbt** [reply]
Stages could be useful for larger groups for sure - the downside being that there's no video.

**Reply by eddiejaoude** [reply]
I think we need video. I have a paid google account, so if we hit the limit of 25 in Discord, I can create Google Meet calls that allow 100

Then yes we could do some on live streams too, 1 per month maybe on a live stream and then more frequently in Discord/Meet

**Reply by naomi-lgbt** [reply]
If you create the Meet link, you don't need to be present for the 100 limit to apply, right?

**Reply by suitupstan** [reply]
Yes.. Eddie doesn't need to be there but just create the meeting and leave the call **(Not use the END button next to leave)**

**Reply by naomi-lgbt** [reply]
Perfect. Last thing I want to do is give Eddie more work. 🙂

**Reply by Vyvy-vi** [reply]
I can volunteer to moderate/create the same.

**Reply by Vyvy-vi** [reply]
Also, 100 is the default limit.
For Google Suite users, it is 250

**Reply by suitupstan** [reply]
Awesome!

**Comment by eddiejaoude** [comment]
I like the idea of removing stale bot and doing some more collaborative 👍
